TaskManagement/src/TaskManagment/functionality/TaskDBHelper.java

```diff
@@ -0,0 +1,474 @@
+package TaskManagerProject;
+
+import utils.CollectionUtils;
+import utils.MailUtils;
+
+import java.sql.*;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+
+public class TaskDBHelper {
```

Comment:
It is better to leave a comment before the starting of each method that describes the method